docs/quickstart.md

```diff
@@ -107,3 +107,56 @@ To uninstall an instance and delete all the managed Kubernetes resources:
 ```shell
 timoni -n test delete podinfo --wait
 ```
+
+## Bundling instances
+
+For deploying complex applications to production, it is recommended to use
+Timoni [Bundles](bundles.md).
+
+A Timoni Bundle is a CUE file for defining a group of instances together
+with their values and module references.
+
+The following is an example of a Bundle that defines a Redis master-replica cluster
+and a podinfo instance connected to the Redis instance.
+
+```cue
+bundle: {
+    apiVersion: "v1alpha1"
+    instances: {
+        redis: {
+            module: {
+                url:     "oci://ghcr.io/stefanprodan/modules/redis"
+                version: "7.0.10"
+            }
+            namespace: "podinfo"
+            values: maxmemory: 256
+        }
+        podinfo: {
+            module: url:     "oci://ghcr.io/stefanprodan/modules/podinfo"
+            module: version: "6.3.5"
+            namespace: "podinfo"
+            values: caching: {
+                enabled:  true
+                redisURL: "tcp://redis:6379"
+            }
+        }
+    }
+}
+```
+
+To apply the above bundle on a cluster, save the file as `podinfo.bundle.cue` and run:
+
+```shell
+timoni bundle apply -f podinfo.bundle.cue
+```
+
+To list the instances created from the bundle:
```

Comment:
This is a bit of a misdirection, because the text says "list things created from the bundle" but the command does that only by coincidence -- both instances happen to be in the namespace mentioned, and evidently nothing else is). This example command doesn't generalise. What if there were instances from some other bundle in that namespace? or, the bundle deployed to more than one namespace?

I don't think this is a serious problem but it might give people the wrong idea, or confuse them. Perhaps the text could be "To list the instances in the namespace used by the bundle:" or similar.

Reply:
Made a likewise comment to Stefan in private, and we both had the same ideas around options to improve this beyond phrasing:

1. Make defining a name in a bundle required, which can then be used as a label on the backing Secret. This could then be used to filter the output of the command based on this label.
2. Allow supplying a bundle file, which is then used to filter the output. But this seems to be the lesser option, given not every user may have access to the source (or is a UI).

Reply:
A bundle can contain instances in different namespaces, if we go with labeling, to find them all would be something like:

``
timoni ls --bundle=my-bundle --all-namespaces
``

Reply:
I have replaced listing with deletion in the quickstart. I'll add back listing once we have the label in place. Thanks @squaremo and @hiddeco for the feedback!